packages/venia-ui/lib/components/Header/__tests__/__snapshots__/cartTrigger.spec.js.snap

```diff
@@ -4,7 +4,7 @@ exports[`Cart counter displays 99+ when items quantity is more than 99 1`] = `
 Array [
   <div>
     <button
-      aria-label="Toggle mini cart. You have 100 items in your cart."
+      aria-label="Toggle mini cart. You have {count} items in your cart."
```

Comment:
Would we not expect for the test runner to convert the {count} variable into the actual integer here?

Reply:
I guess this comment got lost, would be good to understand if the snapshots could contain the replacement patterns or if they should be rendered correctly. As we're mainly writing unit tests I would imagine we'd wanna mock our `react-intl` whenever we do not intend for it to translate.